### DISCUSSION TITLE
createCardToken

### DISCUSSION THREAD

**Discussion by Claudionpico** [mercadopago/sdk-js#140] [General]
Tengo un error al crear createCardToken, probe configuranto cn httpS://localhost pero sigue igual 

Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://api.mercadopago.com') does not match the recipient window's origin ('http://localhost:3000').

**Comment by AdaltonLeite** [comment]
Hola Claudio, ¿cómo estás? 

¿Podrías compartir con nosotros cómo estás haciendo tu integración? Este error no es común, tenemos que ver si falta alguna pieza de código.
